### PR TITLE
Build universal binaries for release

### PR DESCRIPTION
**Pull request by thii**
Closes https://github.com/yonaskolb/XcodeGen/issues/994.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,17 +9,18 @@ CURRENT_PATH = $(PWD)
...
 RELEASE_TAR = $(REPO)/archive/$(VERSION).tar.gz
 SHA = $(shell curl -L -s $(RELEASE_TAR) | shasum -a 256 | sed 's/ .*//')
+SWIFT_BUILD_FLAGS = --disable-sandbox -c release --arch arm64 --arch x86_64
 
 .PHONY: install build uninstall format_code brew release
 
 install: build
 	mkdir -p $(PREFIX)/bin
-	cp -f .build/release/$(EXECUTABLE_NAME) $(INSTALL_PATH)
+	cp -f $(shell swift build $(SWIFT_BUILD_FLAGS) --show-bin-path)/$(EXECUTABLE_NAME) $(INSTALL_PATH)
```
Comment (yonaskolb) [inline Makefile:18]: TIL about `--show-bin-path` 👍

**Comment by yonaskolb** [comment]
This is great, thank you @thii. Could you rebase off or merge in master and then add a changelog entry (since you'll get a conflict in the changelog otherwise)

**Comment by thii** [comment]
@yonaskolb Thanks. Rebased and added a changelog entry. I added it to the "Internal" section, but let me know if that isn't the right place.

**Review by yonaskolb — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (toshi0383) [reply Makefile:18]: Also TIL about `shell swift ...`👀

**Comment by yonaskolb** [comment]
This is unfortunately causing issues with Homebrew https://github.com/Homebrew/homebrew-core/pull/74854

**Pull request merged**